gradebook/server
# 🔥 remove knex-migrator

Testing:

1. Run `scripts/prepare-4.0-migration.sh` to rename the existing knex-migrator tables
2. Start the server. Init migrations should automatically run, and no data should be lost.

This PR requires **2** reviewers.

It might be useful to go commit-by-commit, but it also might not be
